templates/app/templates/frontend/src/store/index.js

```diff
@@ -54,15 +45,14 @@ export default new Vuex.Store({
     },
     async accountSignIn({ commit }, { mnemonic }) {
       return new Promise(async (resolve, reject) => {
-        const wallet = await Secp256k1Pen.fromMnemonic(mnemonic);
-        const address = wallet.address("cosmos");
+        const wallet = await Secp256k1Wallet.fromMnemonic(mnemonic);
+        const { address } = wallet;
```

Comment:
This is a private API that might break at any time. Better get the account via `getAccounts` (from the the `OfflineSigner` interface):

```js
const [{ address }] = await wallet.getAccounts();

// or less fancy

const address = (await wallet.getAccounts())[0].address;
```